avatar/avatar.css

```diff
@@ -0,0 +1,29 @@
+:root {
+  --avatar-bg-color: #c2b4cf;
+}
+
+.avatar {
+  width: 150px;
+  margin: auto;
+  background: var(--avatar-bg-color);
+}
+
+.avatar img {
+  width: 150px;
+  height: 150px;
+}
+
+.avatar-1 {
+  padding: 1.5rem;
+}
+
+.avatar-2 {
+  padding: 1.5rem;
+  border-radius: 50%;
+}
+
+.avatar-3 {
+  padding: 1.5rem;
+  border: 2px solid blueviolet;
+  border-radius: 0.3rem;
+}
```

Comment:
and just a suggestion try to create different sizes of the avatar like 
avatar-circle-sm, avatar-circle-md, avatar-circle-lg